contents/docs/self-host/deploy/configuration.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Posthog Clickhouse Chart Configuration
+sidebarTitle: Configuration
```

Comment:
Seems strange to have a configuration title outside of the Configure top-level section. Can we qualify this so it feels less out of place? `Chart Configuration`, maybe?

```suggestion
sidebarTitle: Chart Configuration
```

Reply:
Just to note that I was planning on looking into both of these and merging updating them too, but wanted to land this initial diff first.